[debug] [2024-08-21T06:36:57.740Z] ----------------------------------------------------------------------
[debug] [2024-08-21T06:36:57.742Z] Command:       /usr/local/bin/node /usr/local/bin/firebase deploy
[debug] [2024-08-21T06:36:57.743Z] CLI Version:   13.14.2
[debug] [2024-08-21T06:36:57.743Z] Platform:      darwin
[debug] [2024-08-21T06:36:57.743Z] Node Version:  v20.10.0
[debug] [2024-08-21T06:36:57.743Z] Time:          Wed Aug 21 2024 15:36:57 GMT+0900 (Korean Standard Time)
[debug] [2024-08-21T06:36:57.743Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-08-21T06:36:57.745Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2024-08-21T06:36:57.838Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2024-08-21T06:36:57.838Z] > authorizing via signed-in user (eomsa98@gmail.com)
[debug] [2024-08-21T06:36:57.839Z] [iam] checking project na-o-man-bbaf5 for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2024-08-21T06:36:57.839Z] > refreshing access token with scopes: []
[debug] [2024-08-21T06:36:57.840Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2024-08-21T06:36:57.840Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2024-08-21T06:36:57.843Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: getaddrinfo ENOTFOUND firebase-public.firebaseio.com
[warn] ⚠  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2024-08-21T06:36:57.844Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[debug] [2024-08-21T06:36:57.844Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------464620755759052789261915","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[debug] [2024-08-21T06:36:57.844Z] [iam] error while checking permissions, command may fail: FirebaseError: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2024-08-21T06:36:57.845Z] > refreshing access token with scopes: []
[debug] [2024-08-21T06:36:57.845Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2024-08-21T06:36:57.845Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2024-08-21T06:36:57.848Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------167090268559721007320049","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[debug] [2024-08-21T06:36:57.849Z] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[error] 
[error] Error: Assertion failed: resolving hosting target of a site with no site name or target name. This should have caused an error earlier
[debug] [2024-08-21T06:37:11.177Z] ----------------------------------------------------------------------
[debug] [2024-08-21T06:37:11.179Z] Command:       /usr/local/bin/node /usr/local/bin/firebase deploy
[debug] [2024-08-21T06:37:11.180Z] CLI Version:   13.14.2
[debug] [2024-08-21T06:37:11.180Z] Platform:      darwin
[debug] [2024-08-21T06:37:11.180Z] Node Version:  v20.10.0
[debug] [2024-08-21T06:37:11.180Z] Time:          Wed Aug 21 2024 15:37:11 GMT+0900 (Korean Standard Time)
[debug] [2024-08-21T06:37:11.180Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-08-21T06:37:11.182Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2024-08-21T06:37:11.276Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2024-08-21T06:37:11.276Z] > authorizing via signed-in user (eomsa98@gmail.com)
[debug] [2024-08-21T06:37:11.277Z] [iam] checking project na-o-man-bbaf5 for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2024-08-21T06:37:11.277Z] > refreshing access token with scopes: []
[debug] [2024-08-21T06:37:11.278Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2024-08-21T06:37:11.278Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2024-08-21T06:37:32.475Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------079502692717896643306705","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[debug] [2024-08-21T06:37:32.476Z] [iam] error while checking permissions, command may fail: FirebaseError: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2024-08-21T06:37:32.477Z] > refreshing access token with scopes: []
[debug] [2024-08-21T06:37:32.478Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2024-08-21T06:37:32.478Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2024-08-21T06:37:32.483Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: getaddrinfo ENOTFOUND firebase-public.firebaseio.com
[warn] ⚠  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2024-08-21T06:37:32.485Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[debug] [2024-08-21T06:37:32.486Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------290827785173286327995795","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[debug] [2024-08-21T06:37:32.486Z] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[error] 
[error] Error: Assertion failed: resolving hosting target of a site with no site name or target name. This should have caused an error earlier
[debug] [2024-08-21T06:37:36.155Z] ----------------------------------------------------------------------
[debug] [2024-08-21T06:37:36.157Z] Command:       /usr/local/bin/node /usr/local/bin/firebase deploy
[debug] [2024-08-21T06:37:36.158Z] CLI Version:   13.14.2
[debug] [2024-08-21T06:37:36.158Z] Platform:      darwin
[debug] [2024-08-21T06:37:36.158Z] Node Version:  v20.10.0
[debug] [2024-08-21T06:37:36.158Z] Time:          Wed Aug 21 2024 15:37:36 GMT+0900 (Korean Standard Time)
[debug] [2024-08-21T06:37:36.158Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-08-21T06:37:36.160Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2024-08-21T06:37:36.253Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2024-08-21T06:37:36.253Z] > authorizing via signed-in user (eomsa98@gmail.com)
[debug] [2024-08-21T06:37:36.253Z] [iam] checking project na-o-man-bbaf5 for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2024-08-21T06:37:36.254Z] > refreshing access token with scopes: []
[debug] [2024-08-21T06:37:36.254Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2024-08-21T06:37:36.254Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2024-08-21T06:37:36.258Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: getaddrinfo ENOTFOUND firebase-public.firebaseio.com
[warn] ⚠  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2024-08-21T06:37:36.259Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[debug] [2024-08-21T06:37:36.259Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------639501883281247470439200","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[debug] [2024-08-21T06:37:36.259Z] [iam] error while checking permissions, command may fail: FirebaseError: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2024-08-21T06:37:36.260Z] > refreshing access token with scopes: []
[debug] [2024-08-21T06:37:36.260Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2024-08-21T06:37:36.260Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2024-08-21T06:37:36.264Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------722990955733153175576412","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[debug] [2024-08-21T06:37:36.264Z] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[error] 
[error] Error: Assertion failed: resolving hosting target of a site with no site name or target name. This should have caused an error earlier
